CMakeLists.txt

```diff
@@ -269,6 +269,7 @@ add_subdirectory(src/engine)
 target_precompile_headers(engine PRIVATE ${PRECOMPILED_HEADER_FILES_ENGINE})
 add_subdirectory(src/index)
 add_subdirectory(src/util)
+add_subdirectory(benchmark)
```

Comment:
The infrastructure has a lot of files. Maybe it's time, to move the infrastructure into a sub folder of `benchmark`, so that the actual benchmarks are easier to find later.

Reply:
Done, see [8e6962b](https://github.com/ad-freiburg/qlever/pull/860/commits/8e6962b1c5e1b038e245139e7ddd4e5629ccfa06).

Reply:
That is fine, nothing to do here.